pynest/nest/raster_plot.py

```diff
@@ -121,9 +116,21 @@ def from_file(fname, **kwargs):
 
 def from_file_pandas(fname, **kwargs):
     """Use pandas."""
-    if isinstance(fname, (list, tuple)):
+    if isinstance(fname, str):
+        fname = fname
+        dataFrame = pandas.read_csv(
+            fname, sep='\s+', lineterminator='\n',
+            header=None, index_col=None,
+            skipinitialspace=True)
+        data = dataFrame.values
+
+    elif isinstance(fname, (list, tuple)):
         data = None
         for f in fname:
+            if not isinstance(f, str):
+                print >> sys.stderr, 'List entry is not a python2 string'
+                return None
+
             if data is None:
                 dataFrame = pandas.read_csv(
```

Comment:
And shouldn't this be:
````
for f in fname:
   dataFrame = panda.read_csv(...)
   newdata = dataFrame.values
   if data is None: data = newdata
   else: data = numpy.concatenate(...)
````

Currently it looks to me like this would only read the first fname.